rescheduler/rescheduler.go

```diff
@@ -112,6 +123,11 @@ func main() {
 				if len(criticalPods) > 0 {
 					for _, pod := range criticalPods {
 						glog.Infof("Critical pod %s is unschedulable. Trying to find a spot for it.", podId(pod))
+						k8sApp := "unknown"
+						if l, found := pod.ObjectMeta.Labels["k8s-app"]; found {
```

Comment:
Is this an actual formalized convention? If so can you point it out? I've seen some cluster bootstrapping tools that don't do this according to this convention, and it would be nice to be able to point at something for reference.